### PR TITLE
[TTAHUB-533] Cleanup delivery data

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

Created a script to alter the data some reports have in the delivery type column in the database, based on mappings in the jira.

## How to test
Imports from staging should have the offending reports. Locate them in your DB, run the script, and then confirm that those reports have been edited appropriately, and that the entire database wasn't mucked about with.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-533

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by jasalisbury — COMMENTED** [review]
I get the following when I run the script, of note changing `null to  {}` (with a blank space). I see the issue for `deliverytype` and `deliverymethod`
```
2022-02-04T19:20:19.005Z AUDIT info: 	changing deliverytype: null to  {}
2022-02-04T19:20:19.005Z AUDIT info: 	changing deliverymethod: Email to Virtual {}
2022-02-04T19:20:19.005Z AUDIT info: 	changing deliverytype: null to  {}
2022-02-04T19:20:19.005Z AUDIT info: 	changing deliverymethod: Email to Virtual {}
2022-02-04T19:20:19.005Z AUDIT info: 	changing deliverytype: null to  {}
2022-02-04T19:20:19.006Z AUDIT info: 	changing deliverymethod: Multi-grantee: Recurring event (Community Practice) to  {}
```

**Comment by thewatermethod** [comment]
I think its just changing null to "", the {} comes from the auditlogger method. Do you think that's an issue? I know that deliverymethod has to be "" not ```null```, but I could make have deliveryType be null alternatively

File: src/tools/updateDeliveryData.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  auditLogger.info(`Updating delivery method...
+  
+  `);
+
+  const deliveryMethods = Object.keys(DELIVERY_DICTIONARY);
+
+  const reports = await ActivityReport.findAll({
+    attributes: ['id', 'deliveryMethod', 'virtualDeliveryType', 'imported'],
+    where: {
+      [Op.and]: [
```
Comment (jasalisbury) [inline src/tools/updateDeliveryData.js:66]: Do we need the `[Op.and]` here? Can it just be
```
where: {
  deliveryMethod: {
...
  }
}
```

File: src/tools/updateDeliveryData.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import { Op } from 'sequelize';
+import { ActivityReport } from '../models';
+import { auditLogger } from '../logger';
+
+const DELIVERY_DICTIONARY = {
+  Email: {
+    deliveryMethod: 'Virtual',
+    virtualDeliveryType: '',
```
Comment (jasalisbury) [inline src/tools/updateDeliveryData.js:8]: In the form if a user selects `Virtual` we make them select a delivery type. Do we need to add additional `deliveryType`s to the form so we can set this field to something?

File: src/tools/updateDeliveryData.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+export default async function updateDeliveryData() {
+  auditLogger.info(`Updating delivery method...
+  
+  `);
+
+  const deliveryMethods = Object.keys(DELIVERY_DICTIONARY);
+
+  const reports = await ActivityReport.findAll({
```
Comment (jasalisbury) [inline src/tools/updateDeliveryData.js:63]: Should we only be updating legacy reports?

File: src/tools/updateDeliveryData.js (hunk 2 above)
Comment (thewatermethod) [reply src/tools/updateDeliveryData.js:8]: spoke to angela today - there isn't enough info to add delivery types to these reports

**Review by jasalisbury — APPROVED** [review]

**Pull request merged**